test/e2e/azure_test.go

```diff
@@ -58,10 +59,17 @@ var _ = Describe("Workload cluster creation", func() {
 		Expect(e2eConfig.Variables).To(HaveKey(capi_e2e.KubernetesVersion))
 		Expect(e2eConfig.Variables).To(HaveKey(capi_e2e.CNIPath))
 
+		clusterName = os.Getenv("CLUSTER_NAME")
+		if clusterName == "" {
+			clusterName = fmt.Sprintf("capz-e2e-%s", util.RandomString(6))
+		}
+		fmt.Fprintf(GinkgoWriter, "INFO: using cluster %s", clusterName)
```

Comment:
can we only include this log line when clusterName is not empty and was provided?

Reply:
I thought this was useful in both cases. If it is Randomly generated it is helpful to have that to be clearly specified in the out.  Maybe "using cluster" is the wrong wording here and should be something like "cluster name is"?

Reply:
Yeah, +1 to "Using cluster name" or "cluster name is"

Reply:
updated to `Cluster name is`